docs/guide/mutating_datasets.rst

```diff
@@ -448,7 +322,7 @@ When garbage collection is called, the files are removed.
 
     files_before = set(store.keys())
 
-    garbage_collect_dataset(store=store_factory, dataset_uuid="another_partitioned_dataset")
+    garbage_collect_dataset(store=store_factory, dataset_uuid=dm.uuid)
```

Comment:
here, did you verify the set difference output below?

Reply:
Yes, works as expected